MRETestBed/Assets/TestBed Assets/Scripts/UserInfo.cs

```diff
@@ -6,15 +6,17 @@
 
 internal class UserInfo : IUserInfo
 {
-    public UserInfo(Guid id)
+    public UserInfo(Guid id, string name)
     {
         Id = id;
+        this.name = name;
     }
 
     public GameObject UserGO { get; set; }
 
     public Guid Id { get; }
 
+    private string name;
```

Comment:
Can you rename `name` so that it's not confused with the public `Name` accessor?